vendor/github.com/openshift-kni/oran-o2ims/api/hardwaremanagement/v1alpha1/node.go

```diff
@@ -26,15 +26,15 @@ type NodeSpec struct {
 }
 
 type BMC struct {
-	// Address contains the URL for accessing the BMC over the network.
+	// The Address contains the URL for accessing the BMC over the network.
```

Comment:
The changes in the vendor folder are from a previous update that updated the API files but did not subsequently do a "go mod vendor". Given that this is unrelated to your change, I'd prefer to see that fixed that in a separate PR. We can also setup the CI test to check the git tree and protect against this happening again

Reply:
agree

Reply:
https://github.com/openshift-kni/oran-o2ims/pull/188